bower.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "perf-widget",
+  "version": "1.0.0",
```

Comment:
It's a version number to get bower working, if i corresponds to anything it is the extension since that is what is using the bower component but since we are not publishing to bower it is meaningless.